src/app/(home)/globe/index.tsx

```diff
@@ -149,6 +202,16 @@ function GlobeInner() {
               }}
             />
           </Source>
+
+          {highlightArea && (
+            <MapPopup
+              id={highlightArea.id}
+              itemType={EPageType.area}
+              label={highlightArea.name}
+              longitude={highlightArea.popupLocation.lng}
+              latitude={highlightArea.popupLocation.lat}
+            />
```

Comment:
@oliverroick I suggest we set `offset` property here to make some space between the mouse pointer to the popup bottom.